src/views/settings.hbs

```diff
@@ -0,0 +1,3 @@
+<h1>This is your family Code:</h1>
```

Comment:
I think you should add something like: 'Please save your family code and share these with your family members'.